banzai_nres/classify.py

```diff
@@ -66,9 +81,5 @@ def do_stage(self, image) -> NRESObservationFrame:
             image.meta['CLASSIFY'] = 0, 'Was this spectrum classified'
```

Comment:
Suggest clarification in the description: '1 if this spectrum was newly classified, 0 if old classification was used.'
Maybe something like this goes in the docs for the code though.